pox-locking/Cargo.toml

```diff
@@ -22,6 +22,7 @@ path = "src/lib.rs"
 clarity = { package = "clarity", path = "../clarity" }
 stacks_common = { package = "stacks-common", path = "../stacks-common" }
 slog = { version = "2.5.2", features = [ "max_level_trace" ] }
+mutants = "0.0.3"
```

Comment:
Same here -- `mutants` is a dev-dependency, or its an optional and off-by-default dependency in all crates.

Reply:
moved it to dev-dependency and on all the other places